pkg/bootstrap/config.go

```diff
@@ -86,8 +86,15 @@ func (cfg Config) toTemplateParams() (map[string]interface{}, error) {
 		option.OutlierLogPath(cfg.Metadata.OutlierLogPath),
 		option.ProvCert(cfg.Metadata.ProvCert),
 		option.DiscoveryHost(discHost),
+		option.Metadata(cfg.Metadata),
```

Comment:
Probably should scope this down to cfg.Metadata.Labels?

Reply:
This is just making it available to the template, should not have any performance impact or anything. The main risk I see would be someone may customize bootstrap and rely on fields we later take away - but we already do not have a strong API here. Are there other concerns?

Reply:
Mainly to be consistent with existing code, no functional concern. All other existing metadata fields are passed in as specific options, but here we are passing in the whole metadata for just labels. But anyway this is just a subjective suggestion, I guess up to @vishalpowar whether to update it or not ;).

Reply:
My initial draft had changes with just Metadata.Labels being passed :) 
But i agree with @howardjohn that passing Metadata in its entirety is quite reasonable.